ext/pcntl/tests/001.phpt

```diff
@@ -25,13 +24,11 @@ function test_exit_signal(){
 	print "\n\nTesting pcntl_wifsignaled....";
 
 	$pid=pcntl_fork();
-    
+
 	if ($pid==0) {
-		sleep(10);
-		exit;
+		posix_kill(posix_getpid(), SIGTERM);
 	} else {
 		$options=0;
-		posix_kill($pid, SIGTERM);
```

Comment:
The test fails 10% of the time when run locally and on [travis-ci](https://api.travis-ci.org/jobs/141183042/log.txt?deansi=true). 

The reason is a race condition on the return of fork. See the commit message for the strace of a good/bad run.